go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/nikandfor/mq-golang-jms20
```

Comment:
The module definition here looks to be for your fork of the project - I think this should actually be `module github.com/ibm-messaging/mq-golang-jms20`.

Then at line 8 of this file this would remove the import for this `mq-golang-jms20` module?

Reply:
Yes, you are right, my fault. I used tools but didn't checked it with my eyes.